cityofaustin/atd-vz-data
# 9498 edit delete note

## Associated issues

Closes cityofaustin/atd-data-tech/issues/9498

## Testing
**URL to test:** 

This can be tested locally or as a Netlify preview https://deploy-preview-1081--atd-vze-staging.netlify.app/#/crashes

**Steps to test:**
1. Click on the Crashes tab and select any crash ID.
2. Add a new note.
3. Edit the note and save it (pencil button then check button).
4. Click on edit but then click the cancel (big X) button to cancel, note should be left unchanged.
5. Click on edit, don't add or change anything, then click the save (check) button. The note should have remained the same.
6.  Delete the note (trash can button).

---
#### Ship list
